CMakeLists.txt

```diff
@@ -250,6 +250,13 @@ if(BUILD_TESTING)
   foreach(executable shptest shputils)
     add_executable(${executable} ${executable}.c)
     target_link_libraries(${executable} PRIVATE ${PACKAGE})
+    if (WIN32 OR CYGWIN)
+      add_custom_command(
+        TARGET ${executable} POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_RUNTIME_DLLS:${executable}> $<TARGET_FILE_DIR:${executable}>
```

Comment:
TARGET_RUNTIME_DLLS is a CMake 3.21 new feature. A if (CMAKE_VERSION VERSION_GRETER_EQUAL 3.21) test should be added (here and in tests/CakeLists.txt)

Reply:
Right. Fixed.